docs/book/src/cronjob-tutorial/testdata/project/test/e2e/e2e_suite_test.go

```diff
@@ -64,24 +66,36 @@ var _ = BeforeSuite(func() {
 	err = utils.LoadImageToKindClusterWithName(projectImage)
 	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "Failed to load the manager(Operator) image into Kind")
 
-	// Setup Prometheus and CertManager before the suite if not skipped
+	// Setup Prometheus and CertManager before the suite if not skipped and if not already installed
 	if !skipPrometheusInstall {
-		_, _ = fmt.Fprintf(GinkgoWriter, "Installing Prometheus Operator...\n")
-		Expect(utils.InstallPrometheusOperator()).To(Succeed(), "Failed to install Prometheus Operator")
+		_, _ = fmt.Fprintf(GinkgoWriter, "Checking for existing installation of Prometheus Operator...\n")
+		isPrometheusOperatorAlreadyInstalled = utils.IsPrometheusOperatorInstalled()
```

Comment:
```suggestion
                By("checking if prometheus is installed already") 
		isPrometheusOperatorAlreadyInstalled = utils.IsPrometheusOperatorInstalled()
```

Reply:
I think we need add a comment to explain why we are doing those checks
Also, we need to use By("") to be able to properly check it in the tests logs